integrations/gateway-api-inference-extension/prometheus_metadata.yaml

```diff
@@ -45,3 +46,50 @@ platforms:
         kind: GAUGE
         value_type: DOUBLE
     install_documentation_url: https://cloud.google.com/stackdriver/docs/managed-prometheus/exporters/inference-optimized-gateway
+  - type: GKE
+    launch_stage: GA
```

Comment:
Is the library with the new metric prefix already live? Should this be GA or should you use HIDDEN first?

Note that the integration config changes and the new dashboard config will roll out as part of two different binary rollouts, so it's easier to set this as HIDDEN, have both roll out, validate that the new dashboard shows up correctly for the new integration (hidden behind an experiment), and then set this to GA.

Reply:
Thanks for catching it. Updated.